miri

```diff
@@ -140,7 +140,7 @@ run|run-debug)
     cargo build $CARGO_BUILD_FLAGS
     find_sysroot
     # Then run the actual command.
-    exec cargo run $CARGO_BUILD_FLAGS "$@"
+    exec cargo run $CARGO_BUILD_FLAGS -- --sysroot "$MIRI_SYSROOT" "$@"
```

Comment:
Oh dang, this will do the wrong thing if `MIRI_SYSROOT` is unset. I will fix that later.